### DISCUSSION TITLE
PP calculations are incorrect when Classic mod isn't being used

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#15681] [Q&A] [answered]
I don't know whether or not this is intentional, but if it isn't I'd imagine that it's because slider ends don't count towards combo and the game thinks that the max combo of the map is higher than it actually is. This also affects the in-game pp counter as well.

Beatmap: https://osu.ppy.sh/beatmapsets/1621891#osu/3311337

Without Classic mod (777 max combo, 229pp):

![osu_2021-11-19_04-38-30](https://user-images.githubusercontent.com/74088315/142610682-f437220e-7df0-463a-83fa-f3acf29efb20.png)

With Classic mod (1320 max combo, 330pp):

![osu_2021-11-19_04-39-28](https://user-images.githubusercontent.com/74088315/142610688-5f3d1bf2-14e1-4e04-a0fe-0134360d2739.png)

**Comment by apollo-dw** [comment, marked as answer]
duplicate of https://github.com/ppy/osu/issues/12542
